src/njoy21/input/DTFR/Card2/Iedit/test/Iedit.test.cpp

```diff
@@ -0,0 +1,34 @@
+#define CATCH_CONFIG_MAIN
+#include "catch.hpp"
+#include "njoy21.hpp"
+
+using namespace njoy::njoy21::input;
+
+SCENARIO( "DTFR iedit values",
+  "[DTFR],[Card2], [Iedit]"){
+
+  GIVEN( "valid iedit values" ){
+    iRecordStream<char> iss1( std::istringstream( "0" ) );
```

Comment:
Why not use the variable names `iss0` and `iss1`? It would more directly imply the correlation to the desired input value.